tox.ini

```diff
@@ -93,12 +93,12 @@ deps =
     # Latest developer version of infrastructure packages.
     devinfra: git+https://github.com/pytest-dev/pytest.git
     devinfra: git+https://github.com/astropy/extension-helpers.git
-    devinfra: git+https://github.com/astropy/pytest-doctestplus.git
+    devinfra: git+https://github.com/scientific-python/pytest-doctestplus.git
```

Comment:
Note: This should go in no matter what. This is an oversight from when we moved pytest-doctestplus to Scientific Python.